### DISCUSSION TITLE
Single file executable for windows and a welcome page with basic keymaps instead of unnamed file.

### DISCUSSION THREAD

**Discussion by debajyoti1990** [SpartanJ/ecode#80] [Ideas]
Basically the title.

**Comment by SpartanJ** [comment]
It will be implemented without a doubt. I'm still thinking of a good welcome page, I have some ideas.
I want to first try to get a definitive icon/logo for the project. I really need some help in that area.

**Reply by debajyoti1990** [reply]
Could you describe the origin of the name 'ecode' ?

**Reply by SpartanJ** [reply]
Same origin as [eepp](https://github.com/SpartanJ/eepp/). The first `e` comes from `entropia`, which means `entropy` in Spanish. So `ecode` is basically `entropia code`.

**Reply by debajyoti1990** [reply]
I have a few ideas based on 2 facts

1. e stands for entropy
2. ecode is written in C++


 can try to post pictures if you're interested.

**Reply by emeraldtip** [reply]
Maybe something like this for the logo:
![image](https://user-images.githubusercontent.com/48155462/224642854-1fc43448-adaf-4ca5-b09d-413c3eff968b.png)
You'd have the text "ecode" in a stylized font just slowly get entropied out of existence with some extra details if required

**Reply by debajyoti1990** [reply]
My ideas

![Ecode idea.jpg](https://user-images.githubusercontent.com/48386861/224655738-506d3a59-0e5a-4d16-9282-1dcebade2831.jpg)

![Ecode 2.jpg](https://user-images.githubusercontent.com/48386861/224656031-36cea8ac-ca22-41e1-8eb1-ea37ca2a96bc.jpg)

**Reply by debajyoti1990** [reply]
Or may be this 

![Ecode3.jpg](https://user-images.githubusercontent.com/48386861/224656839-04cffb14-dde4-4a5f-90f0-6d83e1280bd8.jpg)

**Reply by emeraldtip** [reply]
Maybe this?
![image](https://user-images.githubusercontent.com/48155462/224695622-75753d53-d231-440e-b774-bd33bcbf3a52.png)

As I have stated before:
![image](https://user-images.githubusercontent.com/48155462/224695707-a667929a-72db-4841-b07d-d0ae04328f7e.png)

**Reply by SpartanJ** [reply]
I really appreciate the effort but I think we need a real graphic designer :rofl:

**Reply by emeraldtip** [reply]
This was meant as more of a general design idea brainstorm thing

**Reply by SpartanJ** [reply]
New logo icon:
![ecode-icon-web](https://user-images.githubusercontent.com/650416/226516430-e0e5ab5a-fe4c-4867-b550-1f9adf58190f.svg)

**Comment by emeraldtip** [comment]
Maybe something like the atom welcome screen
![image](https://user-images.githubusercontent.com/48155462/224534910-2c4e4ae2-6c1f-4ede-a04a-da6a790e6d48.png)

**Reply by SpartanJ** [reply]
I really like it. That's a good starting point. What we really need first is some good documentation.

**Reply by SpartanJ** [reply]
Here's a preview of the new Welcome screen (and logo):
![welcome_screen](https://user-images.githubusercontent.com/650416/226516223-159a0fcc-cf8a-491e-ae8f-cc0dc6c8767a.png)
